_im-plugin/ism/policies.md

```diff
@@ -789,6 +789,244 @@ If you want to skip rollovers for an index, set `index.plugins.index_state_manag
    GET _plugins/_ism/explain/log-000001?pretty
    ```
 
+## Sample policies with ISM templates for the alias action
+
+The following sample template policies are for an alias action use case.
+
+#### Rollover an index policy with deletion of indexes when the doc count exceeds a certain value
+
+In the following example, the first job will trigger the rollover action and a new index will be created. Next, another document is added to the two indexes. The new job will then cause the second index to point to the log alias and the older index will be removed due to the alias action.
+
+First, create an ISM policy:
+
+```bash
+PUT /_plugins/_ism/policies/rollover_policy?pretty
+```
+{% include copy-curl.html %}
+
+```json
+{
+  "policy": {
+    "description": "Example rollover policy.",
+    "default_state": "rollover",
+    "states": [
+      {
+        "name": "rollover",
+        "actions": [
+          {
+            "rollover": {
+              "min_doc_count": 1
+            }
+          }
+        ],
+        "transitions": [{
+            "state_name": "alias",
+            "conditions": {
+              "min_doc_count": "2"
+            }
+          }]
+      },
+      {
+        "name": "alias",
+        "actions": [
+          {
+            "alias": {
+              "actions": [
+                {
+                  "remove": {
+                      "alias": "log"
+                  }
+                }
+              ]
+            }
+          }
+        ]
+      }
+    ],
+    "ism_template": {
+      "index_patterns": ["log*"],
+      "priority": 100
+    }
+  }
+}
+```
+
+Next, create an index template to enable the policy on:
+
+```bash
+PUT /_index_template/ism_rollover?
+```
+{% include copy-curl.html %}
+
+```json
+{
+  "index_patterns": ["log*"],
+  "template": {
+   "settings": {
+    "plugins.index_state_management.rollover_alias": "log"
+   }
+ }
+}
+```
+
+Then, change the cluster settings to trigger jobs every minute:
+
+```bash
+PUT /_cluster/settings?pretty=true
+```
+{% include copy-curl.html %}
+
+```json
+{
+  "persistent" : {
+    "plugins.index_state_management.job_interval" : 1
+  }
+}
+```
+
+Next, create a new index:
+
+```bash
+PUT /log-000001
+```
+{% include copy-curl.html %}
+
+```json
+{
+  "aliases": {
+    "log": {
+      "is_write_index": true
+    }
+  }
+}
+```
+
+Finally, add a document to the index to trigger the job:
+
+```bash
+POST /log-000001/_doc
+```
+{% include copy-curl.html %}
+
+```json
+{
+  "message": "dummy"
+}
+```
+
+You can verify these steps using the alias and index API:
+
+```bash
+GET /_cat/indices?pretty
+```
+{% include copy-curl.html %}
+
+```
+GET /_cat/aliases?pretty
+```
+{% include copy-curl.html %}
```

Comment:
If we write this part, better to give the example response, otherwise I don't think we need this part.